### DISCUSSION TITLE
[Data Importer]: Parent import and variants strategy

### DISCUSSION THREAD

**Discussion by KitsuneQki** [pimcore/pimcore#17454] [Q&A / Support] [answered]
Hello! I'm currently looking for some advice with import strategies using data import to create Parent objects and add variants as children, my parent objects need to be filled with general fields like name, and description, but my variants need to be filled with more specific fields like price and attributes, I'm using excel for the data source, is there a way to do this kind of import in only 1 import configuration or are any good practices for do these imports?

**Comment by kingjia90** [comment, marked as answer]
With location strategy as `Find parent` there's a checkbox `as variant` might work
![image](https://github.com/user-attachments/assets/98c57ad5-cee3-48c1-87c9-447e40aabc6f)

Not sure if it's doable in just 1 import configuration, it sounds like there should be 2 configurations, the first one just works on parent and the second one only to the variants once the parents are well defined, even the fact that the data would be different, it would require different mappings 🤔

**Comment by filippo-groppi** [comment]
Yes, I agree with kingjia90, and I confirm that you have to do 2 different import. First on parents the second on variants.
